oracle-23.5.0/listener.ora

```diff
@@ -0,0 +1,12 @@
+# listener.ora Network Configuration File: /opt/oracle/product/23ai/dbhomeFree/network/admin/listener.ora
+# Generated by Oracle configuration tools.
+
+DEFAULT_SERVICE_LISTENER = FREE
+
+LISTENER =
+  (DESCRIPTION_LIST =
+    (DESCRIPTION =
+      (ADDRESS = (PROTOCOL = TCP)(HOST = 0.0.0.0)(PORT = 1521))
```

Comment:
das ist ein bisschen scheisse. er schreibt hier standarmässig die ID des containers rein, das funktioniert dann in neuen container nicht mehr. 

daher kopiere ich dieses file dann manuell mit 0.0.0.0 in den container.